cookbook/configuration/pdo_session_storage.rst

```diff
@@ -111,19 +91,21 @@ a second array argument to ``PdoSessionHandler``:
             # ...
             session.handler.pdo:
                 class:     Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler
+                public:    false
                 arguments:
-                    - "@pdo"
-                    - { 'db_table': 'sessions'}
+                    - "mysql:dbname=mydatabase"
+                    - { db_table: sessions, db_username: myuser, db_password: mypassword }
```

Comment:
The `db_table` option is not needed here (it's not present in the other formats).